_templates/playground/new/react.md.ejs.t

```diff
@@ -0,0 +1,17 @@
+---
+arbitrary: <% pascalName = h.changeCase.pascal(name) %>
+# this file's location depends on whether or not the css option is selected via the prompt
+to: "<%= `static/usage/v${version}/${name.replace('ion-', '')}/${path}/${css ? 'react/main_tsx.md' : 'react.md'}` %>"
+---
+```tsx
+import React from 'react';
+import { <%= pascalName %> } from '@ionic/react';
```

Comment:
This needs `import ./main.css` if CSS is included. We usually put it under here, after a line break.

Reply:
Good catch, thanks!